socorro/external/boto/crashstorage.py

```diff
@@ -391,3 +390,17 @@ def _do_save_processed(boto_connection, processed_crash):
             "crash_report",
             processed_crash_as_string
         )
+
+    @staticmethod
+    def _do_get_unredacted_processed(boto_connection, crash_id, json_object_hook):
+        """Overriding this to change "name of thing" to crash_report"""
+        try:
+            processed_crash_as_string = boto_connection.fetch(crash_id, 'crash_report')
```

Comment:
I wonder if the fact that we're doing this twice means we should just make it a class attribute that can be overridden?

Reply:
This class creates a new data type to store, but the way it does it is this override nonsense. I spent some time thinking about it while working on this PR.

There's another bug for upgrading to boto3. boto3 has a different API than boto, so I was thinking when we did that, it'd be a good opportunity to collapse all the connection contexts into one, nix the keybuilder code, and fix crash data types which should fix the Telemetry crash storage and make it less hacky.

I think that rewrite is in the near future.

Reply:
Having said that, if you think switching it to a class attribute is easy to do, I'd be game for doing it now. Less hacky is less hacky.

Reply:
Nah, it doesn't pass the "will be rewritten shortly" test.